README.md

```diff
@@ -83,9 +83,9 @@ Is your Git repository bursting at the seams?
 
 3.  Change to the directory containing the Git repository that you'd like to analyze, then run
 
-        git sizer [<option>...]
+        git-sizer [<option>...]
 
-    No options are required. You can learn about available options by typing `git sizer -h` or by reading on.
+    No options are required. You can learn about available options by typing `git-sizer -h` or by reading on.
 
     (The above command assumes that you have added `git-sizer` to your `PATH`. If you don't add it to your `PATH`, you need to type its full path and filename to run it; e.g., `/path/to/bin/git-sizer`.)
```

Comment:
This line here ("The above line ...") explains that running `git sizer` *should* work if you have it in your `PATH` and what you have to do if it is not in your path. The reason is that `git foo` knows to look for programs called `git-foo` and run them. This is convenient because you can use `git` options when running the command; for example,

    git -C /path/to/repo foo

will run `git-foo` in the specified path.

Did this not work for you? Do you think this is too confusing?

Reply:
Ah I had no idea git did that.

Reply:
@mhagger not too confusing, that's common for user-defined git scripts (IMO).